e2e/e2e_test.go

```diff
@@ -180,6 +198,59 @@ func verifyTransferToBridgeAccount(setup *setup.Setup, memo string, whbarReceive
 	return *transactionResponse, whbarBalanceBefore
 }
 
+func verifyTokenTransferToBridgeAccount(setup *setup.Setup, memo string, wTokenReceiverAddress common.Address, t *testing.T) (hedera.TransactionResponse, *big.Int) {
+	// Get the wrapped hbar balance of the receiver before the transfer
+	whbarBalanceBefore, err := setup.Clients.WTokenContract.BalanceOf(&bind.CallOpts{}, wTokenReceiverAddress)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Printf("Token balance before transaction: [%s]\n", whbarBalanceBefore)
+	// Get bridge account token balance before transfer
+	receiverBalance, err := hedera.NewAccountBalanceQuery().
+		SetAccountID(setup.BridgeAccount).
+		Execute(setup.Clients.Hedera)
+	if err != nil {
+		fmt.Println(`Unable to query the token balance of the Bridge Account`)
+		t.Fatal(err)
+	}
+	fmt.Println(fmt.Sprintf(`Bridge account Token balance before transaction: [%d]`, receiverBalance.Token[setup.TokenID]))
+
+	// Get the transaction receipt to verify the transaction was executed
+	transactionResponse, err := sendTokensToBridgeAccount(setup, memo)
+	if err != nil {
+		fmt.Println(fmt.Sprintf(`Unable to send Tokens to Bridge Account, Error: [%s]`, err))
+		t.Fatal(err)
+	}
+	transactionReceipt, err := transactionResponse.GetReceipt(setup.Clients.Hedera)
+	if err != nil {
+		fmt.Println(fmt.Sprintf(`Transaction unsuccessful, Error: [%s]`, err))
+		t.Fatal(err)
+	}
+
+	fmt.Println(fmt.Sprintf(`Successfully sent Tokens to bridge account, Status: [%s]`, transactionReceipt.Status))
+
+	// Get bridge account hbar balance after transfer
+	receiverBalanceNew, err := hedera.NewAccountBalanceQuery().
+		SetAccountID(setup.BridgeAccount).
+		Execute(setup.Clients.Hedera)
+	if err != nil {
+		fmt.Println(`Unable to query the token balance of the Bridge Account`)
+		t.Fatal(err)
+	}
+
+	fmt.Println(fmt.Sprintf(`Bridge Account Token balance after transaction: [%d]`, receiverBalanceNew.Token[setup.TokenID]))
+
+	// Verify that the custodial address has received exactly the amount sent
+	amount := receiverBalanceNew.Token[setup.TokenID] - receiverBalance.Token[setup.TokenID]
+	// Verify that the bridge account has received exactly the amount sent
+	if amount != uint64(tokensSendAmount) {
+		t.Fatalf(`Expected to recieve the exact transfer amount of hbar: [%v]`, hBarSendAmount.AsTinybar())
+	}
+
+	return *transactionResponse, whbarBalanceBefore
+}
+
 func sendHbarsToBridgeAccount(setup *setup.Setup, memo string) (*hedera.TransactionResponse, error) {
```

Comment:
`sendHbarsToBridgeAccount` and `sendTokensToBridgeAccount` can be moved in `./clients/hedera.go` file. We will reduce the bloat in the main `e2e_test.go` file that way